examples/with-pagination/src/pages/PaginatedNpmSearch.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+
+import SearchResultPage from './components/SearchResultPage';
```

Comment:
This is just a small nit, but I'd just put both components into a single file so that the pattern of pages & container becomes more obvious :v: